h/views/accounts.py

```diff
@@ -323,13 +332,13 @@ def get_when_not_logged_in(self):
                     _(
                         "We didn't recognize that activation link. "
                         "Have you already activated your account? "
-                        'If so, try <a href="{url}">logging in</a> using the username '
+                        "If so, try logging in using the username "
                         "and password that you provided."
-                    ).format(url=self.request.route_url("login"))
+                    ),
                 ),
                 "error",
             )
-            return httpexceptions.HTTPFound(location=self.request.route_url("index"))
+            return httpexceptions.HTTPFound(location=self.request.route_url("login"))
```

Comment:
This one is after _unsuccessfully_ trying to activate your account (using an invalid or used activation link). It now redirects to the login page not index. But it can't pre-fill the form field

Reply:
Yep. Prevents another dead end